var/spack/repos/builtin/packages/perl-xml-libxml/package.py

```diff
@@ -19,10 +19,13 @@ class PerlXmlLibxml(PerlPackage):
 
     license("GPL-1.0-or-later OR Artistic-1.0-Perl")
 
+    version("2.0210", sha256="a29bf3f00ab9c9ee04218154e0afc8f799bf23674eb99c1a9ed4de1f4059a48d")
+    version("2.0209", sha256="b4a5abbcd689aa2fbbc8b7b45339e961c4984e48108494eb6c282b4748222425")
     version("2.0201", sha256="e008700732502b3f1f0890696ec6e2dc70abf526cd710efd9ab7675cae199bc2")
 
     depends_on("libxml2")
     depends_on("perl-xml-namespacesupport", type=("build", "run"))
     depends_on("perl-xml-sax", type=("build", "run"))
     depends_on("perl-xml-sax-base", type=("build", "run"))
     depends_on("perl-alien-libxml2", type="build")
+    conflicts("%gcc@14", when="@:2.0209")
```

Comment:
Line break for readability, open-ended since presumably gcc@15 won't work either...
```suggestion

    conflicts("%gcc@14:", when="@:2.0209")
```